.storybook/preview.js

```diff
@@ -1,5 +1,5 @@
 import { useGlobals, useParameter } from "@storybook/addons"
-import "../app/main.css"
```

Comment:
@aureentuluvacal i know this is a WIP PR so apologies if you intend to revert this, but curious what motivated this change? i think we probably want to load the `/app/main.css` file

Reply:
Yep! Was gonna revert